### DISCUSSION TITLE
Support for php 7.4

### DISCUSSION THREAD

**Discussion by mcolominas** [spatie/laravel-sitemap#376] [Q&A]
I had this package installed, but when updating the composer now it gives me the following error message:

spatie/laravel-sitemap dev-master requires php ^8.0 -> your php version (7.4.4) does not satisfy that requirement.

What version do I have to specify in composer.json to work with php 7.4?

**Comment by pgrimaud** [comment]
You should add the ^5.9 version for php7.4 compatibility. 

https://packagist.org/packages/spatie/laravel-sitemap#5.9.2
